### PR TITLE
Tutorials

### PR DESCRIPTION
**Pull request by NeelayS**
- Added DML tutorial
- Removed imports from root __init__.py

### CODE REVIEW

**Comment by NeelayS** [comment]
This much of a description of the paper is okay right ?

**Comment by NeelayS** [comment]
Don't know why Travis shows issues with black formatting. I've done it locally.

**Comment by Het-Shah** [comment]
It is failing for another pr as well something is wrong with black.

**Comment by Het-Shah** [comment]
> This much of a description of the paper is okay right ? Then I'll do the rest this way as well

Maybe add the schematic of the algorithm from the paper?

File: docs/usage/tutorials/index.rst [hunk 1]
```diff
@@ -2,6 +2,7 @@ Tutorials
 =========
 
 .. toctree::
-    :maxdepth: 2
+    :maxdepth: 3
```
Comment (Het-Shah) [inline docs/usage/tutorials/index.rst:5]: No need to increase this

Comment (NeelayS) [reply docs/usage/tutorials/index.rst:5]: Oh okay

**Comment by NeelayS** [comment]
The algo snippet for DML is not complete on its own. It refers to equations from earlier in the paper. What should be done ?
I think this will be the case for other papers as well. Should we avoid putting the schematics in the tutorials ?

**Comment by Het-Shah** [comment]
> The algo snippet for DML is not complete on its own. It refers to equations from earlier in the paper. What should be done ?
I think this will be the case for other papers as well. Should we avoid putting the schematics in the tutorials ?

yea true. Can you a diagrammatic snippet from the paper? 

image

This one in particular?

**Comment by NeelayS** [comment]
Hmm, yes, I'll add this one and similar for other algos

**Review by Het-Shah — APPROVED** [review]

**Comment by Het-Shah** [comment]
Merging

**Pull request merged**